tests/run/pure_fused.pxd

```diff
@@ -1,6 +1,9 @@
+import cython
+
 ctypedef fused NotInPy:
     int
     float
 
 cdef class TestCls:
+    @cython.locals(loca = NotInPy)
```

Comment:
There's another one here.

Reply:
It's the same one in a different place - otherwise it'd be broken :) Thanks